chat-sample/tsconfig.json

```diff
@@ -3,15 +3,16 @@
 		"module": "Node16",
 		"target": "ES2022",
 		"lib": [
-			"ES2022"
+			"ES2022",
+			"DOM"
```

Comment:
Why is this needed? This is running in a node process with no DOM. You mentioned console, seems like console methods are available as expected